.circleci/config.yml

```diff
@@ -249,7 +249,7 @@ jobs:
     parameters:
       test_args:
         type: string
-        default: "CLUSTER=0 AOF=0"
+        default: "CLUSTER=0"
```

Comment:
If we're disabling the valgrind-aof job, we should change it's name in circleci to something more appropriate.